cktap/proto.py

```diff
@@ -327,28 +344,37 @@ def get_slot_usage(self, slot, cvc=None):
 
         return (addr, status, here)
 
-    def sign_digest(self, cvc: str, digest: bytes, slot: int=0, subpath: str=None) -> bytes:
+    def sign_digest(self, cvc: str, digest: bytes, slot: int=0, subpath: str=None, bip32_path: str = None) -> bytes:
         """
         Sign 32 bytes digest and return 65 bytes long recoverable signature.
 
         Uses derivation path based on current set derivation on card plus optional
         subpath parameter which if provided, will be added to card derivation path.
         Subpath can only be of length 2 and non-hardened components only.
+        if subpath is specified - use current derivation + derive subpath
+        if bip32_path is specified - subpath is ignored and derivation goes from root
```

Comment:
bip32_path => fullpath or subkey_path ... maybe rename subpath too

Reply:
renamed:
subpath --> sub_deriv
bip32_path --> fullpath

Reply:
renamed sub_deriv back to subpath --> so we do not "fork" other people code